src/agent/onefuzz-agent/src/local/cmd.rs

```diff
@@ -5,17 +5,20 @@ use anyhow::Result;
 use clap::{App, SubCommand};
 
 use crate::local::{
-    common::add_common_config, generic_crash_report, generic_generator, libfuzzer,
-    libfuzzer_coverage, libfuzzer_crash_report, libfuzzer_fuzz, radamsa,
+    common::add_common_config, generic_analysis, generic_crash_report, generic_generator,
+    libfuzzer, libfuzzer_coverage, libfuzzer_crash_report, libfuzzer_fuzz, libfuzzer_merge,
+    radamsa,
 };
 
 const RADAMSA: &str = "radamsa";
 const LIBFUZZER: &str = "libfuzzer";
 const LIBFUZZER_FUZZ: &str = "libfuzzer-fuzz";
 const LIBFUZZER_CRASH_REPORT: &str = "libfuzzer-crash-report";
 const LIBFUZZER_COVERAGE: &str = "libfuzzer-coverage";
+const LIBFUZZER_MERGE: &str = "libfuzzer_merge";
```

Comment:
Use the `-` instead of `_` to match others.

```suggestion
const LIBFUZZER_MERGE: &str = "libfuzzer-merge";
```